scripts/set-image-pull-policy.sh

```diff
@@ -18,8 +18,8 @@ fi
 
 IMAGE_PULL_POLICY=$1
 
-echo IMAGE_PULL_POLICY=$IMAGE_PULL_POLICY
+echo IMAGE_PULL_POLICY="$IMAGE_PULL_POLICY"
 
-sed -i -e "s;\(imagePullPolicy\:\ \).*;\1$IMAGE_PULL_POLICY;" $(git grep -l 'imagePullPolicy' deployments/*.yaml demo/*.yaml)
-
-sed -i -e "s;\(ImagePullPolicy\:\ \).*;\1\"$IMAGE_PULL_POLICY\",;" $(git grep -l 'ImagePullPolicy' pkg/controllers/*/*.go)
```

Comment:
pkg/controllers/*/*.go is not processed in the new version. It doesn't look like a fix fot shellcheck errors to me.

Reply:
it's ok to have it here if it's done intentionally.

Reply:
yes it's intentional